server/src/main/java/chat/engine/Server.java

```diff
@@ -5,20 +5,28 @@
 import java.io.IOException;
 import java.net.ServerSocket;
 import java.net.Socket;
+import java.util.ArrayList;
+import java.util.List;
 
 public class Server {
 
     private static final Logger LOGGER = Logger.getLogger(Server.class);
 
     private static final int PORT = 8080;
 
+    public List<User> clientsOnline = new ArrayList<>();
```

Comment:
What the different between this list and `clientBase` list in 'AuthorizationService'?